adapter/internal/oasparser/operator/operator.go

```diff
@@ -51,7 +51,10 @@ func GetMgwSwagger(apiContent []byte) model.MgwSwagger {
 		if err != nil {
 			logger.LoggerOasparser.Error("Error openAPI unmarsheliing", err)
```

Comment:
is it not required to return this unmarshal err as well?

Reply:
Just a log would suffice in this particular instance as we just need to know that an unmarshalling error occurred. The problem with returning at this point is, this will stop applying the API as a whole. But since it won't be a breaking error for basic functionalities of the API, we decided to not return. @VirajSalaka correct me if I'm wrong.